# Using-Multi-omic-Cancer-Data-to-Find-Ways-to-Improve-the-Treatment-of-Bladder-Cancer

## This is a group project, done by Z. Liu, G. Mizzi, I. Peneva and E. Wershof.

### Abstract
In recent literature on bladder cancer there is call time and time again for better identification of cancer subtypes. Broadly, bladder cancer can be split into two types: muscle invasive and non-muscle invasive. By successfully identifying sub-phenotypes of bladder cancer using multi-omic data, we hope to be better able to personalise cancer care and choose appropriate treatments. To achieve this goal, we analysed the rich data source available to us via The Cancer Genome Atlas,
in particular focussing on Methylation, Gene Expression and Copy Number Variation data. We found evidence supporting current research on using the gene AQP1 as a bladder cancer biomarker. We classified a subgroup of patients with a higher mean age, who had highly methylated genes but better survival prognosis. Further, we identified ways in which multi-omic data could be used to provide more detail than the currently used TNM system. Finally, we performed Multiple Dataset Integration (MDI) to provide a way of combining the different data types.

____
* Report and supplementary materials are in folder Report.
* All codes used to generate the results are in folder src.

## License
All content in this repository is openly licensed with a (license name), which means you're free to use the materials and remix them so long as you **credit the source**.

If you were to use content from this repo in your own work, please attribute the authors with a sentence like: 
> The material is (partially) derived from project "[Using Multi-omic Cancer Data to Find Ways to Improve the Treatment of Bladder Cancer](https://github.com/lzdh/Using-Multi-omic-Cancer-Data-to-Find-Ways-to-Improve-the-Treatment-of-Bladder-Cancer)" done by Z. Liu, G. Mizzi, I. Peneva and E. Wershof .

